### ISSUE TITLE
Mobile responsive support on pattern details pages

### ISSUE THREAD

**Issue by SeanyB** [Shopify/polaris-react#7937]
We should take a closer look at the pattern details page in mobile browser sizes to see if there are optimizations that we need to make.

Having looked at it myself (Sean), I think it's pretty good. But we need a designer eye on it to catch anything I may have missed.

Note: We should prioritize the copy + assets for the remaining patterns before this.

**Comment by tracytees** [comment]
## Taking a look at the mobile formatting, I've noticed some things that could use some adjustments:

![IMG_4049](https://user-images.githubusercontent.com/108495662/210896427-3dbe1521-42bc-4d68-92df-4272ae135795.PNG)

- [ ] The image is quite tight in terms of spacing...it would be great if that could be adjusted

![IMG_4051](https://user-images.githubusercontent.com/108495662/210901090-71b641c8-1fc6-42b3-9d09-b91f67542418.PNG)

- [ ] Is it possible to adjust the table so the divider is more centered? 

![IMG_4050](https://user-images.githubusercontent.com/108495662/210901794-fa57e820-116b-437a-bd85-8da6a5bbac02.PNG)

- Not a big deal but the "Related Resources" takes up a lot of space...is it possible to have them be 2-up rather than one per line? 

![IMG_4052](https://user-images.githubusercontent.com/108495662/210903288-7f199388-1c68-464e-8737-9aa855787e44.PNG)

- [ ] The right edge of the tabbed background looks like it's cut off. It should have the same margin at the left

![IMG_4053](https://user-images.githubusercontent.com/108495662/210904524-64aa9e02-d58e-426b-a71d-98c2c5cd2bb4.PNG)

- [ ] Despite the text being too light in dark mode, the images need a border or more space between each other. As of right now, it looks like everything is one image

**Comment by SeanyB** [comment]
Thanks, @tracytees. 

I think both the Related resources and images blending into each other will be addressed by work @bernardojoaogarcia is already doing in https://github.com/Shopify/polaris/issues/7949. 

However, the other points are new things we missed initially.

>  Is it possible to adjust the table so the divider is more centered?

I think we may want to have this respond by grouping them on top of each other , rather than side-by-side.

@bernardojoaogarcia, @johanstromqvist  - what do you think of these points?

**Comment by johanstromqvist** [comment]
- Agree w Tracy re the need to review padding and margins in general. Reference the "Component page" in most cases.
- Agree on the vertical stacking in tables. Same regarding the two column layout of "Useful to know", we probably want them one column on small screens.
- The outlines of tabbed content should probably not be "wrapping" but rather "side-to-side", i.e. it's not meaningful to spend horizontal real estate the vertical borders.

I also find the hero image that breaks down the pattern and the live code preview to be quite indistinguishable. It should be clear that one is an image and the other is an interactive code render. They would benefit from a visual treatment that makes that clear. I think the ADG make visuals vs code distinctions quite clear. This point is not unique to mobile.

<img width="1196" alt="image" src="https://user-images.githubusercontent.com/38329124/211359234-f59aeea2-370c-4481-862d-aa4d3c438981.png">
<img width="913" alt="image" src="https://user-images.githubusercontent.com/38329124/211359477-1464f375-01e5-4195-96e6-6e24997cb644.png">

**Comment by johanstromqvist** [comment]
@tracytees Do you know the status of this issue? We could edit the top comment and track todos there, like this: 

![Image](https://user-images.githubusercontent.com/38329124/212436339-f3c2148b-5dde-461b-81e0-568887760ea4.png)

**Comment by jesstelford** [comment]
Cross-posting [this comment](https://github.com/Shopify/polaris/issues/8056#issuecomment-1384723322):

> Ensure a 1/3 to 2/3 horizontal layout between the text and images.

How does this behave responsively? At some point, the 1/3 area for text will get too small to be readable, so we'll need to switch to a stacked layout (text above/below the image). That has the impact of making the images full width, which in turn increases their height, meaning they can take up significant vertical space on mobile, so should we limit their height in those cases to something reasonable?

☝️ I think there are going to be a few of these kinds of questions. Not earth shattering decisions that need to be made, but might take a bit to slog through all the different cases for all the different elements on the page.

**Comment by tracytees** [comment]
After pairing with @jesstelford and @gwyneplaine we made some decisions, which are documented in this [Figma page](https://www.figma.com/file/JUKo15oKz1aCcdeNP4UxyE/Reboot-Polaris-Patterns?node-id=4871%3A220949)
**TL;DR**
- Images will be full width: ignoring the page padding
- Tables: Instead of having text side by side, have the description information go under the heading
- Tabs:  full bleed bleed bg and sticky tabs to the top

**Comment by johanstromqvist** [comment]
Sounds good!

**Comment by johanstromqvist** [comment]
@tracytees are the updates shipped?
- If so, could you close the issue?
- If not, could you unassign yourself and move it to `in progress`
- Or if something needs further review, let us know

**Comment by tracytees** [comment]
@johanstromqvist this is now in the hands of @gwyneplaine. I've removed myself.

**Comment by gwyneplaine** [comment]
@johanstromqvist  we've done an initial design pass over this (thanks @tracytees), this is now in the feature branch. I'll ping for a review on staging once we do another "release"
